hail/src/main/scala/is/hail/expr/ir/Env.scala

```diff
@@ -41,18 +39,18 @@ case class BindingEnv[V](
   def deleteEval(name: String): BindingEnv[V] = copy(eval = eval.delete(name))
 
   def bindAgg(name: String, v: V): BindingEnv[V] =
-    copy(agg = Some(agg.get.bind(name, v)))
+    copy(agg = Some(aggOrEmpty.bind(name, v)))
```

Comment:
I wonder if this should be agg.get.bind, too?  What's the case that fails?  You can't bind something in an environment that doesn't exist.

Reply:
oops, must have forgotten to revert. Should work without this change.